docs/source/tutorials/mapping_reads_reference_assembly.md

```diff
@@ -0,0 +1,190 @@
+#  Mapping reads against a reference/assembly
+
+## 1. What is read-mapping?
+
+Read mapping is aligning each of your reads to a/the matching locus on a longer sequence.
```

Comment:
Read mapping is "placing" your reads onto a locus with a matching sequence in a longer sequence.

Reply:
ok